Assets/MRTK/SDK/Features/UX/Scripts/ProgressIndicators/IProgressIndicator.cs

```diff
@@ -44,5 +44,10 @@ public interface IProgressIndicator
         /// </summary>
         Task CloseAsync();
 
+        /// <summary>
+        /// Closes the progress indicator immediately. No fade out animation. All gameObjects for the progress indicator are reset  to the correct values.
+        /// </summary>
+        void CloseImmediate();
```

Comment:
Changing the interface will be a breaking change. Do we feel okay with that for this scenario?

Reply:
I think it's fine for this scenario. This component doesn't have a very large scope and it should be easy to implement for users.